app/scripts/components/dashboard/waterpoints.jsx

```diff
@@ -32,6 +33,10 @@ const WaterPoints = React.createClass({
           )}
         </BoundsMap>
         <ChartsContainer>
+          charts for waterpoints...
+          <MetricStatus metric="54.65" title="chart.title.functional" total="123456"/>
```

Comment:
To pass in real numbers instead of strings, use js expressions -- `metric={54.65}`.
(I know these are just placeholders, so it doesn't really matter)